etc/profile-m-z/tidal-hifi.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for tidal-hifi
+# Description: The web version of Tidal running in electron with hifi support thanks to widevine.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include tidal-hifi.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}.cache/AMD/VkCache
+noblacklist ${HOME}.cache/fontconfig
+noblacklist ${HOME}.cache/mesa_shader_cache
+noblacklist ${HOME}.config/Electron
+noblacklist ${HOME}.config/pulse
+noblacklist ${HOME}.config/tidal-hifi
+
+include disable-common.inc
+include disable-devel.inc
+#include disable-exec.inc # breaks widevine
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+mkdir ${HOME}.cache/AMD/VkCache
+mkdir ${HOME}.cache/fontconfig
+mkdir ${HOME}.cache/mesa_shader_cache
+mkdir ${HOME}.config/Electron
+mkdir ${HOME}.config/pulse
+mkdir ${HOME}.config/tidal-hifi
+whitelist ${HOME}.cache/AMD/VkCache
+whitelist ${HOME}.cache/fontconfig
+whitelist ${HOME}.cache/mesa_shader_cache
+whitelist ${HOME}.config/Electron
+whitelist ${HOME}.config/pulse
+whitelist ${HOME}.config/tidal-hifi
+
+apparmor
+caps.drop all
+netfilter
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp !chroot
+seccomp.block-secondary
+tracelog
+
+disable-mnt
+private-bin chrome-sandbox,tidal-hifi
+private-cache
+private-dev
+private-etc xdg,pulse,dbus-1,machine-id,fonts,hosts,pki,resolv.conf,ssl
+private-opt tidal-hifi
+private-tmp
+
+# dbus-user filter # will break mpris integration
+dbus-user.talk org.freedesktop.Notifications
+dbus-system none
+
+join-or-start tidal-hifi
```

Comment:
Required?

Reply:
I believe join-or-start should be required, a second instance breaks due to the port being in use.